opennms-provision/opennms-provision-shell/src/main/java/org/opennms/netmgt/provision/requisition/command/ShowImport.java

```diff
@@ -96,7 +98,6 @@ public Object execute() throws Exception {
                     System.out.println("\nInterrupted.");
                 } catch (ExecutionException e) {
                     System.out.printf("\nRequisition retrieval failed with: %s\n", e);
-                    e.printStackTrace();
```

Comment:
Can we keep this? The printf statement above won't show the stack.

Reply:
will do.